### PR TITLE
1st milestone issues

### LINKED ISSUE #45: Recieve page: create flow
1) Connect with redux store
2) Generate QRCODE
3) Implement Copy To Clipboard

### PR DESCRIPTION
**Pull request by bodogaii**
#33 TopAppBar on small screens fix
#45 Recieve page: create flow
#39 Send Page: Confirm button on web-mobile layout issue

### CODE REVIEW

File: src/app/components/top-bar/top-bar.scss [hunk 1]
```diff
@@ -23,14 +23,18 @@
...
         color: grey;
+        padding-left: 0px;
     }
 
     .center {
         text-align: center;
     }
 
+    .middle-padding {
+        padding-right: 56px;
```
Comment (krisboit) [inline src/app/components/top-bar/top-bar.scss]: why is this needed ? on small screens it occupies 56px on the right, and if the page title is long like (Recover wallet) `...` will be shown event if there is enough space.

File: src/app/pages/receive/receive.scss [hunk 2]
```diff
@@ -10,10 +10,13 @@
         height: auto;
         max-width: 300px;
         margin: 0 auto;
+        margin-bottom: 16px;
     }
 
     .address {
         word-wrap: break-word;
-        margin-top: 16px;
+        font-size: 1.5rem !important;
```
Comment (krisboit) [inline src/app/pages/receive/receive.scss]: no need for !important here

File: src/app/pages/receive/receive.scss [hunk 3]
```diff
@@ -10,10 +10,13 @@
...
+        color: gray !important;
+        text-align: center !important;
```
Comment (krisboit) [inline src/app/pages/receive/receive.scss]: no need for important here

File: src/app/pages/send/send.component.tsx [hunk 4]
```diff
@@ -67,7 +66,7 @@ export class SendPage extends Component<IProps, IState> {
     public render() {
         return (
             <div class="send-page">
-                <LayoutGrid>
+                <LayoutGrid class="no-padding-top-bottom">
```
Comment (krisboit) [inline src/app/pages/send/send.component.tsx]: why did you remove the padding here ?

File: src/app/components/top-bar/top-bar.scss (hunk 1 above)
Comment (bodogaii) [reply src/app/components/top-bar/top-bar.scss]: This is equivalent with the space required for the "X"  . I added this so I can use center text for the middle part even when we dont have a right section on the top bar. On my 320px × 568px setup in chrome i didnt had problems with the display. I can delete this but then, for a better fixing of the middle section I think I need to remove the center text from it also when there is no right section. I noticed that the center text makes that middle section not alligned. What do you think is best Tibi ?

File: src/app/pages/receive/receive.scss (hunk 2 above)
Comment (bodogaii) [reply src/app/pages/receive/receive.scss]: I will remove that. Thanks

File: src/app/pages/receive/receive.scss (hunk 3 above)
Comment (bodogaii) [reply src/app/pages/receive/receive.scss]: I will remove that. Thanks

File: src/app/pages/send/send.component.tsx (hunk 4 above)
Comment (bodogaii) [reply src/app/pages/send/send.component.tsx]: This indeed could be a mistake, I did it because the confirm button on the 320 × 568px disappears also when you press the advance button. So I did this which indeed doesnt fixes this entirely. I will add back the padding and maybe think of another sollution for the confirm button for this case also ? Maybe a scrollable screen ?

Comment (krisboit) [reply src/app/pages/send/send.component.tsx]: now it seems to be ok ! :)

File: src/app/components/top-bar/top-bar.scss (hunk 1 above)
Comment (krisboit) [reply src/app/components/top-bar/top-bar.scss]: with the latest changes i think it's much better, for now. We can do improvements in future...
